### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by mattias-p**
Some 404s and some 301s

### CODE REVIEW

File: docs/specifications/tests/Nameserver-TP/nameserver07.md [hunk 1]
```diff
@@ -39,5 +39,5 @@ Foundation) Copyright (c) 2013, 2014, 2015, AFNIC Creative Commons Attributi
 4.0 International License
 
 You should have received a copy of the license along with this work. If not,
-seehttp://creativecommons.org/licenses/by/4.0/.
+seehttps://creativecommons.org/licenses/by/4.0/.
```
Comment (matsduf) [inline docs/specifications/tests/Nameserver-TP/nameserver07.md]: see https://creativecommons.org/licenses/by/4.0/

File: docs/specifications/tests/Nameserver-TP/nameserver08.md [hunk 2]
```diff
@@ -50,5 +50,5 @@ Foundation) Copyright (c) 2013, 2014, 2015, AFNIC Creative Commons Attributi
 4.0 International License
 
 You should have received a copy of the license along with this work. If not,
-seehttp://creativecommons.org/licenses/by/4.0/.
+seehttps://creativecommons.org/licenses/by/4.0/.
```
Comment (matsduf) [inline docs/specifications/tests/Nameserver-TP/nameserver08.md]: see https://creativecommons.org/licenses/by/4.0/.

File: docs/specifications/tests/Nameserver-TP/nameserver09.md [hunk 3]
```diff
@@ -45,5 +45,5 @@ Foundation) Copyright (c) 2013, 2014, 2015, AFNIC Creative Commons Attributi
 4.0 International License
 
 You should have received a copy of the license along with this work. If not,
-seehttp://creativecommons.org/licenses/by/4.0/.
+seehttps://creativecommons.org/licenses/by/4.0/.
```
Comment (matsduf) [inline docs/specifications/tests/Nameserver-TP/nameserver09.md]: see https://creativecommons.org/licenses/by/4.0/

File: docs/internal-documentation/maintenance/SystemTesting.md [hunk 4]
```diff
@@ -75,14 +75,14 @@ The set of configurations must include at least:
...
 
          ```sh
          sudo cpanm Zonemaster-Engine-${ENGINE_VERSION}.tar.gz
          ```
 
-      3. Follow the [post-installation sanity check](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md#post-installation-sanity-check) section of the installation guide to the letter.
+      3. Follow the [post-installation sanity check](https://github.com/dotse/zonemaster-engine/blob/master/docs/Installation.md#post-installation-sanity-check) section of the installation guide to the letter.
 
    4. Install Zonemaster Backend
       1. Follow the [dependencies](https://github.com/dotse/zonemaster-backend/blob/master/docs/installation.md#dependencies)
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/internal-documentation/maintenance/SystemTesting.md]: Done.

**Pull request merged**